test/bats/azure-test.bats

```diff
@@ -198,7 +196,7 @@ SLEEP_TIME=1
     assert_success
     # wait for the httpserver cache to be invalidated
     sleep 15
-    run kubectl run crdtest --namespace default --image=wabbitnetworks.azurecr.io/test/notary-image:signed
+    run kubectl run crdtest --namespace default --image=${TEST_REGISTRY}/test/notary-image:signed
```

Comment:
does the `/test/notary-image:signed` exist? I thought it would be `${TEST_REGISTRY}/notation:signed`?

Reply:
thanks for catching! I'll fix it!